src/test/java/org/jabref/logic/importer/util/RenameReviewToCommentTest.java

```diff
@@ -0,0 +1,86 @@
+package org.jabref.logic.importer.util;
+
+import java.util.Collections;
+
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.FieldName;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class RenameReviewToCommentTest {
+
+    private RenameReviewToComment action;
+
+    @BeforeEach
+    void setUp() {
+        action = new RenameReviewToComment();
+    }
+
+    @Test
+    void noFields() {
+        BibEntry entry = createMinimalBibEntry();
+        ParserResult actualParserResult = new ParserResult(Collections.singletonList(entry));
+
+        action.performAction(actualParserResult);
+
+        assertEquals(new ParserResult(Collections.singletonList(entry)).getDatabase().getEntryByKey("Entry1"),
+                actualParserResult.getDatabase().getEntryByKey("Entry1"));
+    }
+
+    @Test
+    void reviewField() {
+        BibEntry actualEntry = createMinimalBibEntry();
+        actualEntry.setField(FieldName.REVIEW, "My Review");
+        ParserResult actualParserResult = new ParserResult(Collections.singletonList(actualEntry));
+
+        BibEntry expectedEntry = createMinimalBibEntry();
+        expectedEntry.setField(FieldName.COMMENT, "My Review");
+
+        action.performAction(actualParserResult);
+
+        assertEquals(new ParserResult(Collections.singletonList(expectedEntry)).getDatabase().getEntryByKey("Entry1"),
+                actualParserResult.getDatabase().getEntryByKey("Entry1"));
+    }
+
+    @Test
+    void commentField() {
+        BibEntry entry = createMinimalBibEntry();
+        entry.setField(FieldName.COMMENT, "My Comment");
+        ParserResult actualParserResult = new ParserResult(Collections.singletonList(entry));
+
+        action.performAction(actualParserResult);
+
+        assertEquals(new ParserResult(Collections.singletonList(entry)).getDatabase().getEntryByKey("Entry1"),
+                actualParserResult.getDatabase().getEntryByKey("Entry1"));
+    }
+
+
+    @Test
+    void reviewAndCommentField() {
+        BibEntry actualEntry = createMinimalBibEntry();
+        actualEntry.setField(FieldName.REVIEW, "My Review");
+        actualEntry.setField(FieldName.COMMENT, "My Comment");
+
+        ParserResult actualParserResult = new ParserResult(Collections.singletonList(actualEntry));
+
+        BibEntry expectedEntry = createMinimalBibEntry();
+        expectedEntry.setField(FieldName.COMMENT, "My Comment\nReview:\nMy Review");
+
+        action.performAction(actualParserResult);
+
+        assertEquals(new ParserResult(Collections.singletonList(expectedEntry)).getDatabase().getEntryByKey("Entry1"),
```

Comment:
why not just `assertEquals(expectedEntry, actualParserResult.getDatabase().getEntryByKey("Entry1"))`?

Reply:
good remark! (: